### PR TITLE
Spectra v2.3.5

### PR DESCRIPTION
**Pull request by RajkiranBagal**
* Global - Removed Backward compatibility of button padding for old users from multiple blocks.
* Global - Updated SVG icons rendering conditions & optimized the loading.
* Fix: Container - Full height and Fixed background Image not working properly on editor.
* Fix: Info-Box - CTA padding not working on responsive devices.
* Fix: Heading - Disabling heading and sub-heading hides the separator.
* Fix: Forms - Using form inside inner blocks, it was showing an error on submit.
* Fix: Background control - Image has wrong units for custom positioning.

### CODE REVIEW

File: src/blocks/advanced-heading/deprecated.js [hunk 1]
```diff
@@ -414,8 +415,12 @@ const deprecated = [
 				{ descText }
 			</div>
 		);
-	}
-}
+		},
+	},
+	{
+		attributes: attributes_v2_3_5,
```
Comment (aapatre) [inline src/blocks/advanced-heading/deprecated.js:421]: I think all looks good but for the naming convention of deprecated markup, we can use the previous release as the name (v2_3_4).

File: blocks-config/forms/class-uagb-forms.php [hunk 2]
```diff
@@ -57,6 +57,49 @@ public function __construct() {
...
+		/**
+		 *  Get the Inner blocks array.
+		 *
+		 * @since 2.3.5
+		 * @access private
+		 *
+		 * @param  array $blocks_array Block Array.
+		 * @param  int   $block_id of Block.
+		 *
+		 * @return array $final_inner_forms_array inner blocks Array.
```
Comment (AbhijitNage123) [inline blocks-config/forms/class-uagb-forms.php:69]: Change to name of a function instead of `$final_inner_forms_array` .

Comment (AbhijitNage123) [reply blocks-config/forms/class-uagb-forms.php:69]: PR for same - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/2699

File: src/blocks/advanced-heading/deprecated.js (hunk 1 above)
Comment (AbhijitNage123) [reply src/blocks/advanced-heading/deprecated.js:421]: PR for same - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/2699

**Review by AbhijitNage123 — APPROVED** [review]

File: blocks-config/forms/class-uagb-forms.php [hunk 3]
```diff
@@ -57,6 +57,49 @@ public function __construct() {
...
+		 *
+		 * @return array $recursive_inner_forms inner blocks Array.
+		 */
+		private function recursive_inner_forms( $blocks_array, $block_id ) {
+			if ( empty( $blocks_array ) ) {
+				return;
+			}
+
+			foreach ( $blocks_array as $blocks ) {
+				if ( ! empty( $blocks ) ) {
```
Comment (patilvikasj) [inline blocks-config/forms/class-uagb-forms.php:77]: We can do `continue` loop when $blocks is empty instead of big if block.

**Review by patilvikasj — CHANGES_REQUESTED** [review]
Please check the comment. Rest looks good to me.

File: blocks-config/forms/class-uagb-forms.php (hunk 3 above)
Comment (AbhijitNage123) [reply blocks-config/forms/class-uagb-forms.php:77]: Rise a PR for same - https://github.com/brainstormforce/ultimate-addons-for-gutenberg/pull/2700

**Review by patilvikasj — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Review by rajmohanreddy774 — APPROVED** [review]

**Review by supriyajjadhav — APPROVED** [review]

File: readme.txt [hunk 4]
```diff
@@ -170,6 +170,15 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 
 == Changelog ==
 
+= 2.3.5 - THURSDAY, 2nd FEBRUARY 2023 =
+* Global - Removed Backward compatibility of button padding for old users from multiple blocks.
+* Global - Updated SVG icons rendering conditions & optimized the loading.
```
Comment (bsf-zanev) [inline readme.txt:175]: These Global Entries should be tagged as improvements.

**Review by bsf-zanev — CHANGES_REQUESTED** [review]

File: readme.txt (hunk 4 above)
Comment (AbhijitNage123) [reply readme.txt]: Resolved

Comment (RajkiranBagal) [reply readme.txt]: Done

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Pull request merged**